doc/TESTS.rst.txt

```diff
@@ -28,51 +28,53 @@ is found in a routine, you should write a new test for that specific
 case and add it to the test suite to prevent that bug from creeping
 back in unnoticed.
 
-To run SciPy's full test suite, use the following::
+To run NumPy's full test suite, use the following::
 
-  >>> import scipy
-  >>> scipy.test()
+  >>> import numpy
+  >>> numpy.test()
 
 or from the command line::
 
   $ python runtests.py
 
-SciPy uses the testing framework from :mod:`numpy.testing`, so all
-the SciPy examples shown here are also applicable to NumPy.  NumPy's full test
-suite can be run as follows::
+NumPy's full test suite can be run as follows::
 
   >>> import numpy
   >>> numpy.test()
 
 The test method may take two or more arguments; the first, ``label`` is a
 string specifying what should be tested and the second, ``verbose`` is an
 integer giving the level of output verbosity. See the docstring for
-numpy.test for details.  The default value for ``label`` is 'fast' - which
+`numpy.test
+<https://github.com/numpy/numpy/blob/master/numpy/_pytesttester.py>`__
+for details.  The default value for ``label`` is 'fast' - which
```

Comment:
While this will work, it would be better to add at the end of the document something like (untested, please check)
```
.. _numpy test:
.. autofunction:: numpy._pytesttester.PytestTester.__call__
```
then you can reference that here with 
```
:ref:`numpy test`
```

Reply:
It's not working, maybe we need to put the link here.

Reply:
made it work in 1920d8c